tests/test-kubernetes.sh

```diff
@@ -30,12 +30,19 @@ kubectl_config() {
 kubectl_deploy() {
     kubeclt_clean
 
+    echo "install Istio"
```

Comment:
Installing software should be added to scripts/install.sh. Is there a reason this is here instead?

Reply:
We have to install Istio after scripts/bx_auth.sh. Since scripts/install.sh executed before bx_auth, I decided to put it in the kubectl_deploy function.